### PR TITLE
added ng-wat behind the scenes proposal

### PR DESCRIPTION
**Pull request by shairez**

### CODE REVIEW

File: ng-wat-behind-the-scenes/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+At the official ng-conf 2015, Shai Reznik gave a super crazy talk named: [ng-wat](https://www.youtube.com/watch?v=M_Wp-2XA9ZU).
+Which made fun of some of the confusing parts of Angular and soon became viral amongst Angular developers.
+In this ng-conf Israel talk, Shai will share his personal story behind the ng-wat.
+Having the opportunity to sit down with the Angular core team and discuss the future plans of Angular,
+Shai will also share his imperssions regarding the next versions and how we can help make it better, with less WATs :)
+
+
+### SESSION METADATA
+Target Audience: Everyone
+Tags: ng-wat, funny
```
Comment (olgakogan) [inline ng-wat-behind-the-scenes/README.md]: duration? :)

Comment (shairez) [inline ng-wat-behind-the-scenes/README.md]: 20 min

Comment (Tzafra) [inline ng-wat-behind-the-scenes/README.md]: *impressions

Comment (shairez) [inline ng-wat-behind-the-scenes/README.md]: Thanks for the fix @Tzafra

**Pull request merged**